lib/experimental/fonts/font-library/font-library.php

```diff
@@ -56,10 +56,18 @@ function wp_register_font_collection( $config ) {
 	}
 }
 
-// @core-merge: This code needs to be removed.
 add_action(
 	'enqueue_block_editor_assets',
 	function () {
 		wp_add_inline_script( 'wp-block-editor', 'window.__experimentalFontLibrary = true', 'before' );
 	}
 );
+
+$default_font_collection = array(
+	'id'             => 'google-fonts',
+	'name'           => 'Google Fonts',
+	'description'    => __( 'Add from Google Fonts. Fonts are copied to and serverd, from your server.' ),
```

Comment:
```suggestion
	'description'    => __( 'Add from Google Fonts. Fonts are copied to and served from your site.' ),
```
Fixes typo and uses more user-friendly (?) "site" rather than "server".